src/transformers/training_args.py

```diff
@@ -478,6 +480,8 @@ class TrainingArguments:
             are also available. See the [Ray documentation](
             https://docs.ray.io/en/latest/tune/api_docs/analysis.html#ray.tune.ExperimentAnalysis.get_best_trial) for
             more options.
+        use_mps_devices (`bool`, *optional*, defaults to `False`):
+            Whether to use Apple Silicon M1 chip based `mps` device.
```

Comment:
Removed "M1" since this is more general. For instance, it still works on their newer M2 machines.

```suggestion
            Whether to use Apple Silicon chip based `mps` device.
```